lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -173,18 +173,18 @@ class TestFieldOverrideMongoPerformance(FieldOverridePerformanceTestCase):
 
     TEST_DATA = {
         # (providers, course_width, enable_ccx): # of sql queries, # of mongo queries, # of xblocks
-        ('no_overrides', 1, True): (26, 7, 19),
-        ('no_overrides', 2, True): (134, 7, 131),
-        ('no_overrides', 3, True): (594, 7, 537),
-        ('ccx', 1, True): (26, 7, 47),
-        ('ccx', 2, True): (134, 7, 455),
-        ('ccx', 3, True): (594, 7, 2037),
-        ('no_overrides', 1, False): (26, 7, 19),
-        ('no_overrides', 2, False): (134, 7, 131),
-        ('no_overrides', 3, False): (594, 7, 537),
-        ('ccx', 1, False): (26, 7, 19),
-        ('ccx', 2, False): (134, 7, 131),
-        ('ccx', 3, False): (594, 7, 537),
+        ('no_overrides', 1, True): (27, 7, 14),
+        ('no_overrides', 2, True): (135, 7, 85),
+        ('no_overrides', 3, True): (595, 7, 336),
+        ('ccx', 1, True): (27, 7, 14),
+        ('ccx', 2, True): (135, 7, 85),
+        ('ccx', 3, True): (595, 7, 336),
+        ('no_overrides', 1, False): (27, 7, 14),
+        ('no_overrides', 2, False): (135, 7, 85),
+        ('no_overrides', 3, False): (595, 7, 336),
+        ('ccx', 1, False): (27, 7, 14),
+        ('ccx', 2, False): (135, 7, 85),
+        ('ccx', 3, False): (595, 7, 336),
     }
```

Comment:
The remaining discrepancy here (I think) is due to clearing out children when binding for a new student. For old-mongo, that's the only block cache we have. For split, we cache at the modulestore layer, so blowing away the child class doesn't actually force new xblock construction.